presto-verifier/src/main/java/com/facebook/presto/verifier/rewrite/QueryRewriteConfig.java

```diff
@@ -31,6 +31,21 @@ public class QueryRewriteConfig
     private QualifiedName tablePrefix = QualifiedName.of("tmp_verifier");
     private Map<String, Object> tableProperties = ImmutableMap.of();
 
+    private boolean reuseTable;
+
+    public boolean isReuseTable()
+    {
+        return reuseTable;
+    }
+
+    @ConfigDescription("If true, reuse the output table of the source query. Otherwise, run the query and write to a temporary shadow table.")
```

Comment:
1.Is it the best effort?
if it is set to true, but the output table is deleted, does the verification fail or follow original routine?

2.Does it only apply to INSERT and CTAS queries? if so, we should specify in the config description

Reply:
1. If the output table is deleted, the verification will fail.
2. Revised.